### PR TITLE
Content adjustmemts for the query parameter section

### PR DESCRIPTION
**Pull request by sichvoge**
According to a short conversation we had, query parameters should treat object types, union types, and arrays in query parameters consistent to headers.

### CODE REVIEW

File: versions/raml-10/raml-10.md [hunk 1]
```diff
@@ -1227,26 +1227,53 @@ When referencing an inner element of a schema, a RAML processor MUST validate an
...
+    properties:
+      employeeNr: integer
+  Teacher:
+    type: [ Person, Employee ]
 ```
 
-Note: Multiple inheritance is only allowed if all Type Expressions are simple object Types.
+In the example above, the type `Teacher` inherits all restrictions from `Person` and `Employee`.
+
+Multiple inheritance is allowed only if the sub-type is still a valid type declaration after inheriting all restrictions from its parent types. Also, it is not allowed to inherit from different kind of primitive types, for example `[ number, string ]`.
```
Comment (usarid) [inline versions/raml-10/raml-10.md]: I think you also need to explicitly state how to treat this statement when you have union types somewhere in the inheritance chain. I would guess the answer is to expand all the unions and and see that at least one of the allowed combinations is a valid type declaration and in particular does not combine different primitive types, but @petrochenko-pavel-a should say definitively.

**Comment by usarid** [comment]
One comment, otherwise LGTM

File: versions/raml-10/raml-10.md (hunk 1 above)
Comment (sichvoge) [inline versions/raml-10/raml-10.md]: How that works for union types has been mentioned in the union types section already. So I'd would not repeat it here.

**Comment by sichvoge** [comment]
It seems something went wrong with that PR as it includes the changes for the query parameter section and the multiple inheritance chapter. I'll close that and create another one that focuses on only the query parameter changes and merge them as it seems there is no mistake.

**Pull request closed without merging**